staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -47,7 +50,11 @@ type resettableCollector interface {
 }
 
 const (
-	APIServerComponent string = "apiserver"
+	APIServerComponent   string = "apiserver"
+	InvalidContentType   string = "invalid-content-type"
+	InvalidIanaCharset   string = "invalid-iana-charset"
+	UnknownContentType   string = "unknown"
+	UnknownRequestMethod string = "unknown"
```

Comment:
We should probably unify these default values between kubelet and apiserver to be nice to our future dashboarding selves. The values aren't "unknown" per se, maybe "invalid" or "disallowed"? I can fix up the kubelet side in #89412

Reply:
I'm okay with 'other'.

Reply:
They're not technically invalid according to the RFC definition, they just don't fall into the well-defined set which we have whitelisted.

Reply:
updated to 'other'